autode/wrappers/G09.py

```diff
@@ -101,7 +105,7 @@ def _get_keywords(calc_input, molecule):
             continue  # Handled after the full set of keywords is set
 
         elif isinstance(keyword, kws.Keyword):
-            kwd_str = keyword.g09 if hasattr(keyword, "g09") else keyword.g16
+            kwd_str = keyword.g09 if getattr(keyword, "g09") else keyword.g16
```

Comment:
I can't figure out why this change was made sorry

Reply:
`Keyword`s now have defined optional attrs for the different EST methods, rather than dynamically adding them. think overall better and way easier to deal with the types